test/include/actions/PrintMaterials.h

```diff
@@ -0,0 +1,33 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef PRINTMATERIALS_H
+#define PRINTMATERIALS_H
+
+#include "Action.h"
+
+class PrintMaterials;
+
+template<>
+InputParameters validParams<PrintMaterials>();
+
```

Comment:
I know this is just a test action, but it still should be documented completely. Please add a Doxygen block here explaining the purpose of this action.